examples/smartcore-ml/README.md

```diff
@@ -0,0 +1,119 @@
+# RISC Zero Rust SmartCore Machine Learning Template
```

Comment:
We have a suggested README format, I'll dig up the link but in essence, in addition to any project specific sections, we suggest this outline:
* Intro
* Use Cases
* Project Organization
* More Resources
* And if you need any project specific sections, those too

This isn't intended to force all readmes into a cookie-cutter, but to the extent you can reorder/reorganize this readme to conform more to that structure, it will be cleaner to read (and maintain) for folks familiar with other examples.

Reply:
Would it be OK to land and adjust this in a followup PR?

Reply:
Yes